tests/test_pygtracker.py

```diff
@@ -18,9 +20,80 @@ def test_version():
 
 # Start tests for generate_course_statistics
 
+
+def test_generate_course_statistics_input_not_exist():
+    tracker = pygtracker.GradeTracker()
+    with raises(ValueError):
+        tracker.generate_course_statistics(course_ids=[500, "511"])
+
+
+def test_generate_course_statistics_input_type1():
```

Comment:
it's better to be explicit here.

```suggestion
def test_generate_course_statistics_course_id_boolean():
```